src/www/views/servers_view/server_connection_indicator/index.ts

```diff
@@ -14,6 +14,11 @@
 import {html, css, LitElement} from 'lit';
 import {customElement, property, state} from 'lit/decorators.js';
 
+// TODO(daniellacosse): file import type definitions
+// eslint-disable-next-line
+// @ts-ignore
+import circle from '../../../assets/circle.png';
```

Comment:
Let's make sure this is loading on Cordova and Electron.

Reply:
electron:
![Screen Shot 2022-05-23 at 15 04 17](https://user-images.githubusercontent.com/3759828/169889693-99333164-7c48-409d-ba80-a7acf10ac75e.png)

Reply:
...waiting on an android build... (i'm using the artifacts! haha)

Reply:
![Screen Shot 2022-05-23 at 15 04 17](https://user-images.githubusercontent.com/3759828/169902131-fa9f728d-169e-49c2-8b9c-eb99d0ce03a4.png)

Cordova - however, I've encountered a couple issues I must address. Will put this in draft for the moment.

Reply:
Issues addressed! Turns out I just had a bad copy of outline in my emulator - fine now:
<img width="418" alt="Screen Shot 2022-05-24 at 16 38 22" src="https://user-images.githubusercontent.com/3759828/170127928-10d126cd-214a-44b0-8a17-d1c094a4889f.png">